cmake/Modules/FindTPLROCTHRUST.cmake

```diff
@@ -0,0 +1,2 @@
+FIND_PACKAGE(rocthrust REQUIRED)
+KOKKOS_CREATE_IMPORTED_TPL(ROCTHRUST INTERFACE LINK_LIBRARIES roc::rocthrust)
```

Comment:
Do we still have the https://github.com/arborx/ArborX/pull/652 issue where the exported rocThrust package potentially adds flags to compile for all GPU architecture it knows about?

Reply:
At least for the version shipped with ROCm 5.7, it works fine. I see only one architecture.

Reply:
We need to find out for sure because IIRC compile time blows up